src/main/webapp/WEB-INF/mutation.textproto

```diff
@@ -58,5 +58,44 @@ mutation {
     token_name: "e.js"
   }
 }
+mutation {
+  type: ADD_NODE
+  start_node: "I"
+}
+mutation {
+  type: CHANGE_TOKEN
+  start_node: "I"
+  token_change {
+    type: ADD_TOKEN
+    token_name: "d.js"
+    token_name: "f.js"
+  }
+}
+reason: "Changing tokens of node B"
+}
+mutation {
+mutation {
+  type: DELETE_EDGE
+  start_node: "A"
+  end_node: "C"
+}
+reason: "Removing edge AC"
+}
+mutation {
+mutation {
+  type: CHANGE_TOKEN
+  start_node: "B"
+  token_change {
+    type: DELETE_TOKEN
+    token_name: "d.js"
+  }
+}
+reason: "Changing tokens of node B"
+}
+mutation {
+mutation {
+  type: DELETE_NODE
+  start_node: "I"
+}
 reason: "Changing tokens of node B"
```

Comment:
This is kind of my fault because I was too lazy to update the reason but can we put more informative strings here?